packages/grid/_modules_/grid/hooks/features/focus/useGridFocus.ts

```diff
@@ -80,6 +90,48 @@ export const useGridFocus = (apiRef: GridApiRef): void => {
     }));
   }, [logger, setGridState]);
 
+  const handleCellMouseUp = React.useCallback(
+    (params: GridCellParams) => {
+      const { cell } = apiRef.current.getState().focus;
+      if (!cell) {
+        return;
+      }
+
+      if (params.id === cell.id && params.field === cell.field) {
```

Comment:
Can we have a mismatch with the cell if we don't check both the `id` and the `field`?

Reply:
Yes, this handler is called when any cell is clicked. I just want `insideFocusedCell.current` to be true when the currently focused cell is clicked.